src/dual-alb-fargate-svc.ts

```diff
@@ -5,7 +5,13 @@ import * as cdk from '@aws-cdk/core';
 import { BaseFargateService, BaseFargateServiceProps } from './main';
 
 
-export interface DualAlbFargateServiceProps extends BaseFargateServiceProps {};
+export interface DualAlbFargateServiceProps extends BaseFargateServiceProps {
+  /**
+   * The load balancer idle timeout, in seconds.
+   * @default - 60.
+  */
+  readonly albidleTimeout?: cdk.Duration;
```

Comment:
```suggestion
  readonly idleTimeout?: cdk.Duration;
```

Let's just name it `idleTimeout` and the description should explain everything.